src/cleanup-after-each.js

```diff
@@ -0,0 +1,6 @@
+module.exports = async function cleanAfterEach() {
```

Comment:
I'd rather this file not exist. Could we just put this code in the `cleanup-after-each.js` file?

Reply:
you probably want this file to be tested / compiled / validated though now.

Reply:
I suppose it's fine. The thing is that it's not going to be included in the UMD because it's not imported into the main export, but that's actually not a big deal because I don't think many people use the UMD anyway.

Reply:
Kinda weird that is called `cleanup-after-each` when it doesn't do that though 🙁

Reply:
Good point, will rename

Reply:
Done!